backend/app/app.healthcheck.js

```diff
@@ -0,0 +1,16 @@
+import { createTerminus } from "@godaddy/terminus";
+
+function onHealthCheck() {
+    // checks if the system is healthy,
```

Comment:
I looked for things to check but I didn't find anything relevant for now, just check if app is up (but in the future we can check here a connection with redis or dns checks)